modules/04 Advanced Javascript/Assignments/9/load_images.js

```diff
@@ -0,0 +1,27 @@
+picsLinksContainer = document.getElementById("pics-links-container");
+picsLinksContainer.innerText = 'Loading Images...';
+
+fetch("https://picsum.photos/list")
+
+.then( data => { 
+    data.json()
+    
+    
+.then( json => {
+    picsLinksContainer.innerHTML = "";
+
+    for(let i in json) {
+        const imageElem = document.createElement("img");
+        const linkElem = document.createElement("a");
+        linkElem.href = json[i]["post_url"];
+        linkElem.target = '_blank';
+        imageElem.src = `https://picsum.photos/200/300?image=${json[i]["id"]}`
+        linkElem.appendChild(imageElem);
+        picsLinksContainer.appendChild(linkElem);
+    }
+})
+
+})
+.catch( error => {
+    console.log(error);
+})
```

Comment:
your answer doesn't solve the question. please read the instructions and try again. if you still don't understand the requirements please let me know